diff --git a/crates/node/.cargo/config.toml b/crates/node/.cargo/config.toml
index 5196ec6..3c8471f 100644
--- a/crates/node/.cargo/config.toml
+++ b/crates/node/.cargo/config.toml
@@ -5,6 +5,11 @@ linker = "aarch64-linux-musl-gcc"
 rustflags = ["-C", "target-feature=-crt-static"]
 [target.armv7-unknown-linux-gnueabihf]
 linker = "arm-linux-gnueabihf-gcc"
+[target.loongarch64-unknown-linux-gnu]
+linker = "loongarch64-linux-gnu-gcc"
+[target.loongarch64-unknown-linux-musl]
+linker = "loongarch64-linux-musl-gcc"
+rustflags = ["-C", "target-feature=-crt-static"]
 # Statically link Visual Studio redistributables on Windows builds
 [target.x86_64-pc-windows-msvc]
 rustflags = ["-C", "target-feature=+crt-static"]
diff --git a/crates/node/package.json b/crates/node/package.json
index 40ab706..f469839 100644
--- a/crates/node/package.json
+++ b/crates/node/package.json
@@ -12,17 +12,13 @@
     "binaryName": "tailwindcss-oxide",
     "packageName": "@tailwindcss/oxide",
     "targets": [
-      "armv7-linux-androideabi",
-      "aarch64-linux-android",
-      "aarch64-apple-darwin",
       "aarch64-unknown-linux-gnu",
       "aarch64-unknown-linux-musl",
       "armv7-unknown-linux-gnueabihf",
-      "x86_64-unknown-linux-musl",
-      "x86_64-unknown-freebsd",
-      "i686-pc-windows-msvc",
-      "aarch64-pc-windows-msvc",
-      "wasm32-wasip1-threads"
+      "loongarch64-unknown-linux-gnu",
+      "loongarch64-unknown-linux-musl",
+      "x86_64-unknown-linux-gnu",
+      "x86_64-unknown-linux-musl"
     ],
     "wasm": {
       "initialMemory": 16384,
diff --git a/scripts/version-packages.mjs b/scripts/version-packages.mjs
index 1641d54..37365a0 100644
--- a/scripts/version-packages.mjs
+++ b/scripts/version-packages.mjs
@@ -26,6 +26,8 @@ const syncedWorkspaces = new Map([
       'crates/node/npm/linux-arm-gnueabihf',
       'crates/node/npm/linux-arm64-gnu',
       'crates/node/npm/linux-arm64-musl',
+      'crates/node/npm/linux-loong64-gnu',
+      'crates/node/npm/linux-loong64-musl',
       'crates/node/npm/linux-x64-gnu',
       'crates/node/npm/linux-x64-musl',
       'crates/node/npm/wasm32-wasi',
